WordPress/Classes/ViewRelated/Reader/Sorting/ReaderSortingOptionSheetViewController.swift

```diff
@@ -76,9 +76,11 @@ class ReaderSortingActionSheetOptionControl: ClosureControl {
         label.isAccessibilityElement = false
 
         accessibilityIdentifier = option.identifier
-        accessibilityLabel = option.sortingOption.accessibilityLabel
-        accessibilityHint = option.sortingOption.accessibilityHint
-        accessibilityValue = option.sortingOption.accessibilityValue
+        accessibilityLabel = NSLocalizedString("Sorting option", comment: "Accessibility label for sorting option")
```

Comment:
Since VoiceOver speaks in this order:
- label
- value
- traits
- hint
I made it so that label is generic text "Sorting option", value gets to be "Recent" or "Popular", and I use the hint to say if the option is selected or not.